client/src/components/Styles/LayoutStyles.jsx

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+
+
+
+const PrimaryText = styled.div`
+
+`
+
+const SecondaryText = styled.div`
+
+`
+const theme = {
+  colors: {
+    primary: 'blue',
+    secondary: 'steelblue',
+    background: '	#f0f8ff',
+    text: '#5A5A5A',
+  },
+  fonts: {
+    main: '"Helvetica Neue", sans-serif',
+  },
+};
+
+const Container = styled.div`
+  display: grid;
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 20px;
+  background-color: ${(props) => props.theme.colors.background};
+  color: ${(props) => props.theme.colors.text};
+  font-family: ${(props) => props.theme.fonts.main};
+`;
+
+const Section = styled.section`
+  margin-bottom: 40px;
+`;
+
+const Heading1 = styled.h1`
```

Comment:
Do we want to put Heading1 and Heading2 here also as div? I'm not so sure since I'm using primary and secondary colors on non header components. Or we can just leave it as headers and I can align my colors with the theme colors. That's also easy to approach